document/core/exec/instructions.rst

```diff
@@ -1630,7 +1630,9 @@ Control Instructions
 
 5. Let :math:`L` be the label whose arity is :math:`m` and whose continuation is the end of the |TRY| instruction.
 
-4. :ref:`Execute <exec-catchn>` the administrative instruction :math:`\CATCHN_m` by :ref:`entering <exec-instr-seq-enter>` the block :math:`\instr_1^\ast` with label :math:`L`.
+6. Let :math:`H` be the exception handler whose arity is :math:`m` and whose continuation is the beginning of :math:`\instr_2^\ast`.
+
+7. :ref:`Install <exec-install-handler>` the exception handler `H` for the block :math:`\val^n~\instr_1^\ast` with label :math:`L`.
```

Comment:
I'd separate this into installing the handler and entering the block, corresponding to the two separate administrative instructions it reduces to:
```suggestion
7. :ref:`Enter <exec-handler-enter>` the exception handler `H`.

8. :ref:`Enter <exec-instr-seq-enter>` the block :math:`\val^n~\instr_1^\ast` with label :math:`L`.
```